<!DOCTYPE html>
<!--[if IE 8]> <html lang="en" class="ie8"> <![endif]-->  
<!--[if IE 9]> <html lang="en" class="ie9"> <![endif]-->  
<!--[if !IE]><!--> <html lang="en"> <!--<![endif]-->  
<head>
    <title>Bobby Ranjan</title>
    <!-- Meta -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Personal Website - Bobby Ranjan">
    <meta name="author" content="Bobby Ranjan">
    <meta name="keywords" content="Bobby Ranjan, BobbyRanjan, Bobby Ranjan, Website, Resume, Developer, Portfolio, Singapore, NTU, Nanyang Technological University">
    <link rel="shortcut icon" href="favicon.ico">  
    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,400italic,300italic,300,500italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'>
    <!-- Global CSS -->
    <link rel="stylesheet" href="assets/plugins/bootstrap/css/bootstrap.min.css">   
    <!-- Plugins CSS -->
    <link rel="stylesheet" href="assets/plugins/font-awesome/css/font-awesome.css">
    <link rel="stylesheet" href="assets/plugins/academicons/css/academicons.min.css">
    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-mfizz/2.3.0/font-mfizz.css">
    <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/master/devicon.min.css">
    
    <!-- Theme CSS -->  
    <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">
    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!--    Vertical Timeline -->
    <link href='http://fonts.googleapis.com/css?family=Droid+Serif|Open+Sans:400,700' rel='stylesheet' type='text/css'>

	<link rel="stylesheet" href="vertical-timeline/css/reset.css">
    <!-- CSS reset -->
	<link rel="stylesheet" href="vertical-timeline/css/style.css">
    <!-- Resource style -->
	<script src="vertical-timeline/js/modernizr.js"></script> 
    <!-- Modernizr -->
</head> 

<body>
    <div class="wrapper">
        <div class="sidebar-wrapper">
            <div class="profile-container">
                <img class="profile img-circle" src="assets/images/Me_Edited.png" alt="" style="width: 150px;height: 150px" />
                <h1 class="name">Bobby Ranjan</h1>
                <h3 class="tagline">Student | Researcher | Nerd</h3>
            </div><!--//profile-container-->
            
            <div class="contact-container container-block">
                <ul class="list-unstyled contact-list">
                    <li class="email"><i class="fa fa-envelope"></i><a href="mailto: bbbranjan@gmail.com">bbbranjan@gmail.com</a></li>
                    <li class="twitter"><i class="fa fa-twitter"></i><a href="https://twitter.com/bbbranjan" target="_blank">bbbranjan</a></li>
                    <li class="linkedin"><i class="fa fa-linkedin"></i><a href="http://linkedin.com/in/bobbyranjan" target="_blank">/in/bobbyranjan</a></li>
                    <li class="github"><i class="fa fa-github"></i><a href="https://github.com/bbbranjan" target="_blank">bbbranjan</a></li>
                    <li class="location"><i class="fa fa-map-marker"></i><a href="https://www.google.com.sg/maps/place/Singapore" target="_blank">Singapore</a></li>
                </ul>
            </div><!--//contact-container-->
            <div class="languages-container container-block">
                <h2 class="container-block-title">Languages</h2>
                <ul class="list-unstyled interests-list">
                    <li>English <span class="lang-desc">(Native)</span></li>
                    <li>Hindi <span class="lang-desc">(Novice)</span></li>
                    <li>French <span class="lang-desc">(Novice)</span></li>
                </ul>
            </div><!--//interests-->
            
            <div class="interests-container container-block">
                <h2 class="container-block-title">Interests</h2>
                <ul class="list-unstyled interests-list">
                    <li>Science &amp; Technology</li>
                    <li>Entrepreneurship</li>
                    <li>Football</li>
                    <li>Music</li>
                </ul>
            </div><!--//interests-->
            
        </div><!--//sidebar-wrapper-->
        
        <div class="main-wrapper">
            
            <section class="section summary-section">
                <h2 class="section-title"><i class="fa fa-user"></i>About Me</h2>
                <div class="summary">
                    <p>Hi! I'm <strong> Bobby Ranjan</strong>, an incoming predoctoral fellow at EMBL Rome. Born in Scotland, raised in India, educated in Singapore and currently working in Italy, I am an ambitious student-researcher in the field of computational biology.</p>
                    <div class="text-center" style="margin-top:30px">
                        <button type="button" class="btn btn-lg btn-default" onclick="window.location.href = 'Resume - Bobby Ranjan.pdf'"><i class="fa fa-file-pdf-o"></i> Resume</button>
                    </div>
                </div><!--//summary-->
            </section><!--//section-->
            
            
            <section class="section education-section">
                <h2 class="section-title"><i class="fa fa-university"></i>Education</h2>
                
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="degree">Predoctoral Fellow, Hackett Group</h3>
                            <div class="time">2021 - </div>
                        </div><!--//upper-row-->
                    </div><!--//meta-->
                    <div class="details">
                        <h4></h4>
                    </div><!--//details-->
                    <div class="university">European Molecular Biology Laboratory, Rome</div>
                </div><!--//item-->
                
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="degree">Bachelor of Engineering (B.E.) in Computer Engineering </h3>
                            <div class="time">2014 - 2018</div>
                        </div><!--//upper-row-->
                    </div><!--//meta-->
                    <div class="degree-minor">
                        Minor in Entrepreneurship <br> 
                        Minor in Life Sciences
                    </div><!--//degree-minor-->
                    <div class="university">Nanyang Technological University, Singapore</div>
                </div><!--//item-->
            </section><!--//section-->
            
            <section class="section experiences-section">
                <h2 class="section-title"><i class="fa fa-book"></i>Selected Publications</h2>
                
                <div class="item">
                    
                    <button type="button" class="collapsible"> <div class="meta">
                        
                        <div class="upper-row">
                            <h3 class="job-title" style="width: 80%"><a href="https://www.nature.com/articles/s41467-021-26085-2", target="_blank">DUBStepR is a scalable correlation-based feature selection method for accurately clustering single-cell data </a></h3>
                            <div class="authors">Bobby Ranjan, Wenjie Sun, Jinyu Park, Kunal Mishra, Florian Schmidt, Ronald Xie, Fatemeh Alipour, Vipul Singhal, Ignasius Joanito, Mohammad Amin Honardoost, Jacy Mei Yun Yong, Ee Tzun Koh, Khai Pang Leong, Nirmala Arul Rayan, Michelle Gek Liang Lim and Shyam Prabhakar</div>
                            <div class="time">March 2021</div>
                        </div><!--//upper-row-->
                        <div class="company">Nature Communications</div>
                    </div><!--//meta--> </button> 
                    
                    <div class="details content">
                        
                        <h4>Abstract</h4>
                        <p>
                            Feature selection (marker gene selection) is widely believed to improve clustering accuracy, and is thus a key component of single cell clustering pipelines. Existing feature selection methods perform inconsistently across datasets, occasionally even resulting in poorer clustering accuracy than without feature selection. Moreover, existing methods ignore information contained in gene-gene correlations. Here, we introduce DUBStepR (Determining the Underlying Basis using Stepwise Regression), a feature selection algorithm that leverages gene-gene correlations with a novel measure of inhomogeneity in feature space, termed the Density Index (DI). Despite selecting a relatively small number of genes, DUBStepR substantially outperformed existing single-cell feature selection methods across diverse clustering benchmarks. Additionally, DUBStepR was the only method to robustly deconvolve T and NK heterogeneity by identifying disease-associated common and rare cell types and subtypes in PBMCs from rheumatoid arthritis patients. DUBStepR is scalable to over a million cells, and can be straightforwardly applied to other data types such as single-cell ATAC-seq. We propose DUBStepR as a general-purpose feature selection solution for accurately clustering single-cell data.
                        </p>
                    </div><!--//details-->
                </div><!--//item-->
                
                <div class="item">
                    <button type="button" class="collapsible">
                        <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title" style="width: 80%"><a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab632/6329577?login=true", target="_blank">RCA2: a scalable supervised clustering algorithm that reduces batch effects in scRNA-seq data</a></h3>
                            <div class="authors"> Florian Schmidt*, Bobby Ranjan*, Quy Xiao Xuan Lin, Vaidehi Krishnan, Ignasius Joanito, Mohammad Amin Honardoost, Zahid Nawaz, Prasanna Nori Venkatesh, Joanna Tan, Nirmala Arul Rayan, Sin Tiong Ong, Shyam Prabhakar</div>
                            <div class="time">July 2021</div>
                        </div><!--//upper-row-->
                        <div class="company">Nucleic Acids Research</div>
                    </div><!--//meta-->
                    </button>
                    <div class="details content">
                        
                        <h4>Abstract</h4>
                        
                        <p>
                            The transcriptomic diversity of cell types in the human body can be analysed in unprecedented detail using single cell (SC) technologies. Unsupervised clustering of SC transcriptomes, which is the default technique for defining cell types, is prone to group cells by technical, rather than biological, variation. Compared to de-novo (unsupervised) clustering, we demonstrate using multiple benchmarks that supervised clustering, which uses reference transcriptomes as a guide, is robust to batch effects and data quality artifacts. Here, we present RCA2, the first algorithm to combine reference projection (batch effect robustness) with graph-based clustering (scalability). In addition, RCA2 provides a user-friendly framework incorporating multiple commonly used downstream analysis modules. RCA2 also provides new reference panels for human and mouse and supports generation of custom panels. Furthermore, RCA2 facilitates cell type-specific QC, which is essential for accurate clustering of data from heterogeneous tissues. We demonstrate the advantages of RCA2 on SC data from human bone marrow, healthy PBMCs and PBMCs from COVID-19 patients. Scalable supervised clustering methods such as RCA2 will facilitate unified analysis of cohort-scale SC datasets. <br /> <br />
                        
                            <b>Availability:</b> RCA2 is implemented in R and is available on <a href="https://github.com/prabhakarlab/RCAv2">GitHub</a>.
                        </p>
                    </div><!--//details-->
                </div><!--//item-->

                <div class="item">
                    <button type="button" class="collapsible">
                        <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title" style="width: 80%"><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04028-4", target="_blank">scConsensus: combining supervised and unsupervised clustering for cell type annotation in single-cell RNA-seq data </a></h3>
                            <div class="authors">Bobby Ranjan*, Florian Schmidt*, Wenjie Sun, Jinyu Park, Mohammad Amin Honardoost, Joanna Tan, Nirmala Arul Rayan and Shyam Prabhakar</div>
                            <div class="time">April 2021</div>
                        </div><!--//upper-row-->
                        <div class="company">BMC Bioinformatics</div>
                    </div><!--//meta-->
                    </button>
                    <div class="details content">
                        
                        <h4>Abstract</h4>
                        <p>
                            <b>Background:</b> Clustering is a crucial step in the analysis of single-cell data. Clusters identified in an unsupervised manner are typically annotated to cell types based on differentially expressed genes. In contrast, supervised methods use a reference panel of labelled transcriptomes to guide both clustering and cell type identification. Supervised and unsupervised clustering approaches have their distinct advantages and limitations. Therefore, they can lead to different but often complementary clustering results. Hence, a consensus approach leveraging the merits of both clustering paradigms could result in a more accurate clustering and a more precise cell type annotation. <br />
                        
                            <b>Results:</b> We present scConsensus, an R framework for generating a consensus clustering by (i) integrating the results from both unsupervised and supervised approaches and (ii) refining the consensus clusters using differentially expressed (DE) genes. The value of our approach is demonstrated on several existing single-cell RNA sequencing datasets, including data from sorted PBMC sub-populations.<br />
                        
                            <b>Conclusions:</b> scConsensus combines the merits of unsupervised and supervised approaches to partition cells with better cluster separation and homogeneity, thereby increasing our confidence in detecting distinct cell types. scConsensus is freely available on <a href="https://github.com/prabhakarlab/scConsensus">GitHub</a>.
                        </p>
                    </div><!--//details-->
                </div><!--//item-->
                
                <div class="item">
                    <button type="button" class="collapsible">
                        <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title" style="width: 80%"><a href="https://www.nature.com/articles/s41588-020-0636-z", target="_blank">Lineage-dependent gene expression programs influence the immune landscape of colorectal cancer</a></h3>
                            <div class="time">May 2020</div>
                        </div><!--//upper-row-->
                        <div class="authors">Hae-Ock Lee, Yourae Hong, Dr. Hakki Etlioglu, Yong Beom Cho, ..., Bobby Ranjan, ..., Shyam Prabhakar, Sabine Tejpar, Woong-Yang Park</div>
                        <div class="company">Nature Genetics</div>
                    </div><!--//meta-->
                    </button>
                    <div class="details content">
                        
                        <h4>Abstract</h4>
                        <p>
                        Immunotherapy for metastatic colorectal cancer is effective only for mismatch repair-deficient tumors with high microsatellite instability that demonstrate immune infiltration, suggesting that tumor cells can determine their immune microenvironment. To understand this cross-talk, we analyzed the transcriptome of 91,103 unsorted single cells from 23 Korean and 6 Belgian patients. Cancer cells displayed transcriptional features reminiscent of normal differentiation programs, and genetic alterations that apparently fostered immunosuppressive microenvironments directed by regulatory T cells, myofibroblasts and myeloid cells. Intercellular network reconstruction supported the association between cancer cell signatures and specific stromal or immune cell populations. Our collective view of the cellular landscape and intercellular interactions in colorectal cancer provide mechanistic information for the design of efficient immuno-oncology treatment strategies.
                        </p>
                    </div><!--//details-->
                </div><!--//item-->
                
<!--
                <div class="item">
                    <button type="button" class="collapsible">
                        <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title"><a href="http://hdl.handle.net/10356/73997", target="_blank">Detection of functional and topological modules in protein interaction networks</a></h3>
                            <div class="time">May 2018</div>
                        </div>
                        <div class="authors">Bobby Ranjan</div>
                        <div class="company">Final Year Project, Nanyang Technological University</div>
                    </div>
                    </button>
                    <div class="details content">
                        
                        <h4>Abstract</h4>
                        <p>
                            Mapping the complete set of protein and gene interactions in the human cell has been a goal of the biological community for nearly two decades, since the first human genome was sequenced. To this end, computational approaches have been studied in depth to allow functional annotation of protein interactions. In this project, we explored the potential of using four common module detection algorithms - stochastic block model, Louvain method, modified Louvain (incremental Louvain) method and link community - in order to detect functional modules of for protein interaction networks. We implemented these algorithms in a Cytoscape application for users to run on their respective networks. Using this application, we conducted a comparative study of the algorithms to understand their applicability in protein function annotation and determine how close topological modules of protein-interaction networks are to their functional modules.
                        </p>
                    </div>
                </div>
-->
                
                <div class="item">
                    <button type="button" class="collapsible"> 
                        <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title"><a href="https://bmcsystbiol.biomedcentral.com/articles/10.1186/s12918-018-0529-2", target="_blank">Composite Mathematical Modeling of Calcium Signaling behind Neuronal Cell Death in Alzheimer's Disease</a></h3>
                            <div class="time">April 2018</div>
                        </div><!--//upper-row-->
                        <div class="authors">Bobby Ranjan, Ket Hing Chong and Zheng Jie</div>
                        <div class="company">BMC Systems Biology - Volume 12 (The Sixteenth Asia-Pacific Bioinformatics Conference)</div>
                    </div><!--//meta-->
                    </button>
                    <div class="details content">
                        
                        <h4>Abstract</h4>
                        <p>
                            <b>Background:</b> Alzheimer's disease (AD) is a progressive neurological disorder, recognized as the most common cause of dementia affecting people aged 65 and above. AD is characterized by an increase in amyloid metabolism, and by the misfolding and deposition of β-amyloid oligomers in and around neurons in the brain. These processes remodel the calcium signaling mechanism in neurons, leading to cell death via apoptosis. Despite accumulating knowledge about the biological processes underlying AD, mathematical models to date are restricted to depicting only a small portion of the pathology. <br />
                            <b>Results: </b> Here, we integrated multiple mathematical models to analyze and understand the relationship among amyloid depositions, calcium signaling and mitochondrial permeability transition pore(PTP)-related cell apoptosis in AD. The model was used to simulate calcium dynamics in the absence and presence of AD. In the absence of AD, i.e. without β-amyloid deposition, mitochondrial and cytosolic calcium level remains in the low resting concentration. However, our in silico simulation of the presence of AD with the β-amyloid deposition, shows an increase in the entry of calcium ions into the cell and dysregulation of Ca<sup>2+</sup> channel receptors on the Endoplasmic Reticulum. This composite model enabled us to make simulation that is not possible to measure experimentally. <br />
                            <b>Conclusions:</b> Our mathematical model depicting the mechanisms affecting calcium signaling in neurons can help understand AD at the systems level and has potential for diagnostic and therapeutic applications.
                        </p>
                    </div><!--//details-->
                </div><!--//item-->
            </section>
            
            <section class="section experiences-section">
                <h2 class="section-title"><i class="fa fa-briefcase"></i>Experience</h2>
<!--
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Co-founder and CTO</h3>
                            <div class="time">January 2019 - Present</div>
                        </div>
                        <div class="company">PGMed</div>
                    </div>
                    <div class="details">
                        PGMed is an online application that provides home-based post-graduate medical examination training in India
                        <ul>
                            <li>Worked with external vendors to design, develop and test web application    </li>
                        </ul>  
                    </div>
                </div>
-->
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Bioinformatics Specialist</h3>
                            <div class="time">August 2018 - Present</div>
                        </div><!--//upper-row-->
                        <div class="company">Prabhakar Lab, Genome Institute of Singapore</div>
                    </div><!--//meta-->
                    <div class="details">
                        <ul>
                            <li>Developing algorithms for cell type identification in single-cell data</li>
                        </ul>  
                    </div><!--//details-->
                </div><!--//item-->
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Software Design Engineer Intern</h3>
                            <div class="time">May - August 2017</div>
                        </div><!--//upper-row-->
                        <div class="company">BitTitan</div>
                    </div><!--//meta-->
                    <div class="details">
                        <ul>
                            <li>Built customer-facing license consumption report for all BitTitan products</li> 
                            <li>Conducted tech feasibility analysis to improve BitTitan’s reporting capacity</li>
                            <li>Built code analysis tool to clean up database references across codebase</li>
                        </ul>  
                    </div><!--//details-->
                </div><!--//item-->
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Technology Analyst Intern</h3>
                            <div class="time">August - December 2016</div>
                        </div><!--//upper-row-->
                        <div class="company">Bank of America, Merrill Lynch (Singapore)</div>
                    </div><!--//meta-->
                    <div class="details">
                        <ul>
                            <li>Worked on the payments processing and payments testing development teams</li> 
                            <li>Redesigned database logging using a queueing mechanism with the help of Apache ActiveMQ and Java Spring Framework</li>
                            <li>Also built an application to help onboard new testers onto the testing platform, using Java, AngularJS and SQL</li>
                        </ul>  
                    </div><!--//details-->
                </div><!--//item-->
                
<!--
                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Summer Intern</h3>
                            <div class="time">May - July 2016</div>
                        </div>
                        <div class="company">Asia Risk Transfer Solutions (ARTS), Singapore</div>
                    </div>
                    <div class="details">
                        <p>Asia Risk Transfer Solutions - a software and technological solutions startup that aims to help the insurance industry and governments create and manage risk transfer products, known as Insurance for the Masses, for developing communities in Asia.</p>
                        <ul>
                            <li>Worked in a small team of 6 on Android application development for the company</li>
                            <li>Developed 2 MVP Android applications for different user groups</li>
                            <li>Enhanced server-side API of the enterprise web application</li>
                            <li>Tech stack included Java, Android and Django</li>
                        </ul>
                    </div>
                </div>
-->
            </section>
            
            <section class="section projects-section">
                <h2 class="section-title"><i class="fa fa-coffee"></i>Conferences &amp; Workshops</h2>
                <div class="intro">
                </div><!--//intro-->
                <div class="item">
                 <ul>
                     <li>Intelligent Systems in Molecular Biology 2021 - Virtual</li>
                     <li>Human Cell Atlas Asia 2020 - Virtual</li>
                     <li>Intelligent Systems in Molecular Biology 2020 - Virtual</li>
                     <li>Cell Symposia 2019 - Singapore</li>
                     <li>Single Cell Analyses Meeting 2019 - Cold Spring Harbor Laboratory, New York, USA</li>
                    </ul>   
                </div><!--//item-->
            </section><!--//section-->
<!--            <section class="skills-section section">
                <h2 class="section-title"><i class="fa fa-cog fa-3x fa-spin"></i>Skills</h2>
                <div class="skillset">
                    <div class="item text-center">
                        
                        <h3 class="level-title"><button type="button" class="btn btn-lg btn-default"> <i class="fa fa-html5" style="font-size:17px"></i> HTML5</button></h3>
                        
                        <h3 class="level-title"><button type="button" class="btn btn-lg btn-default"> <i class="fa fa-css3" style="font-size:17px"></i> CSS3</button></h3>
                        
                        <h3 class="level-title"><button type="button" class="btn btn-lg btn-default"><i class="icon-javascript-alt" style="font-size:17px"></i>JavaScript</button></h3>
                        
                        <h3 class="level-title"> <button type="button" class="btn btn-lg btn-default"><i class="icon-php-alt" style="font-size:17px"></i> PHP</button> </h3>
                        
                    </div>
                    <div class="item text-center">
                        
                        <h3 class="level-title"> <button type="button" class="btn btn-lg btn-default"><i class="icon-java-bold" style="font-size:17px"></i> Java</button> </h3>
                        <h3 class="level-title"> <button type="button" class="btn btn-lg btn-default"><i class="icon-python" style="font-size:17px"></i> Python</button> </h3>
                        <h3 class="level-title"> <button type="button" class="btn btn-lg btn-default"><i class="devicon-csharp-plain" style="font-size:17px"></i> C-Sharp</button> </h3>
                        <h3 class="level-title"> <button type="button" class="btn btn-lg btn-default"><i class="devicon-mysql-plain" style="font-size:17px"></i> MySQL</button></h3>
                    </div>
                    <div class="item text-center">

                        <h3 class="level-title"><button type="button" class="btn btn-lg btn-default"> <i class="devicon-bootstrap-plain"></i> Bootstrap</button></h3>
                        
                        <h3 class="level-title"><button type="button" class="btn btn-lg btn-default"> <i class="icon-angular" style="font-size:20px"></i> AngularJS</button></h3>
                        
                        <h3 class="level-title"><button type="button" class="btn btn-lg btn-default"><i class="icon-nodejs" style="font-size:20px"></i> Node.js</button></h3>
                        
                        <h3 class="level-title"> <button type="button" class="btn btn-lg btn-default"><i class="icon-mongodb" style="font-size:20px"></i> MongoDB</button> </h3>    
                    
                    <h3 class="level-title"> </h3>    
                    <h3 class="level-title"> </h3>
                    <h3 class="level-title"> </h3>
                    <h3 class="level-title"> </h3>
                    </div>
            </div>
        </section>-->
        </div>    
        </div><!--//main-body-->
    <footer class="footer">
        <div class="text-center">
                <!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can check out other license options via our website: themes.3rdwavemedia.com */-->
            <small class="copyright">Built with <i class="fa fa-heart"></i> by Bobby Ranjan, using the <a href = "https://startbootstrap.com/theme/resume", target="_blank">Start Bootstrap resume template </a></small>
        </div><!--//container-->
    </footer><!--//footer-->
 
    <!-- Javascript -->          
    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <script src="vertical-timeline/js/main.js"></script> <!-- Resource jQuery -->
    <script type="text/javascript" src="assets/plugins/bootstrap/js/bootstrap.min.js"></script>
    <!-- custom js -->
    <script type="text/javascript" src="assets/js/main.js"></script>
    
    <script>
var coll = document.getElementsByClassName("collapsible");
var i;

for (i = 0; i < coll.length; i++) {
  coll[i].addEventListener("click", function() {
    this.classList.toggle("active");
    var content = this.nextElementSibling;
    if (content.style.display === "block") {
      content.style.display = "none";
    } else {
      content.style.display = "block";
    }
  });
}
</script>
    
</body>
</html> 


<!-- <section class="section ">
                <h2 class="section-title"><i class="fa fa-heart"></i>A Day in my life (Ideally)</h2>
                <section id="cd-timeline" class="cd-container">
                    <div class="cd-timeline-block">
                        <div class="cd-timeline-img cd-picture">
                            <img src="vertical-timeline/img/cd-icon-sun.svg" alt="Picture">
                        </div> <!-- cd-timeline-img

                        <div class="cd-timeline-content">
                            <h2>Wake up</h2>
                            <p>Always the hardest part of the day</p>
                            <span class="cd-date"><s>6:30</s>  <s>6:40</s>  <s>6:50</s>  
                            7:00 am</span>
                        </div> <!-- cd-timeline-content 
                    </div> <!-- cd-timeline-block 
                    <div class="cd-timeline-block">
                        <div class="cd-timeline-img cd-movie">
                            <img src="vertical-timeline/img/cd-icon-gym.svg" alt="Movie">
                        </div> <!-- cd-timeline-img

                        <div class="cd-timeline-content">
                            <h2>Attack the Gym</h2>
                            <p>Because exercise in the morning is the best thing that could happen to you</p>
                            <span class="cd-date">8:30 am</span>
                        </div> <!-- cd-timeline-content 
                    </div> <!-- cd-timeline-block 
                    <div class="cd-timeline-block">
                        <div class="cd-timeline-img cd-movie">
                            <img src="vertical-timeline/img/cd-icon-work.svg" alt="Movie">
                        </div> <!-- cd-timeline-img 

                        <div class="cd-timeline-content">
                            <h2>Go to work</h2>
                            <p>Jot down a list of things to do during the day</p>
                            <span class="cd-date">10:00 am</span>
                        </div> <!-- cd-timeline-content
                    </div> <!-- cd-timeline-block
                    <div class="cd-timeline-block">
                        <div class="cd-timeline-img cd-movie">
                            <img src="vertical-timeline/img/cd-icon-lunch.svg" alt="Movie">
                        </div> <!-- cd-timeline-img 
                        <div class="cd-timeline-content">
                            <h2>Lunch</h2>
                            <p>My opportunity to talk to people outside work environment</p>
                            <span class="cd-date">12:30 pm</span>
                        </div> <!-- cd-timeline-content 
                    </div> <!-- cd-timeline-block 
                    <div class="cd-timeline-block">
                        <div class="cd-timeline-img cd-movie">
                            <img src="vertical-timeline/img/cd-icon-leave-work.svg" alt="Movie">
                        </div> <!-- cd-timeline-img 

                        <div class="cd-timeline-content">
                            <h2>Leave office</h2>
                            <p>Complete my work for the day, and leave for home</p>
                            <span class="cd-date">6:00 pm</span>
                        </div> <!-- cd-timeline-content 
                    </div> <!-- cd-timeline-block 
                    <div class="cd-timeline-block">
                        <div class="cd-timeline-img cd-movie">
                            <img src="vertical-timeline/img/cd-icon-soccer.svg" alt="Movie">
                        </div> <!-- cd-timeline-img 

                        <div class="cd-timeline-content">
                            <h2>Football (5-a-side)</h2>
                            <p>The beautiful game completes a day of hard work</p>
                            <span class="cd-date">8:30 pm</span>
                        </div> <!-- cd-timeline-content
                    </div> <!-- cd-timeline-block 
	           </section> <!-- cd-timeline 
            </section> -->
                